force-app/main/default/classes/RecipeTriggerHandler_Test.cls

```diff
@@ -0,0 +1,164 @@
+@isTest
+public class RecipeTriggerHandler_Test {
+   @testSetup 
+    public static void setupMethod(){
+        List<Recipe__c> testDraftRecipes = new List<Recipe__c>();
+        integer noOfDraftRecipes = 0;
+        while(noOfDraftRecipes < 150) {
+            Recipe__c draftRecipe = new Recipe__c();
+            draftRecipe.Name = 'Test Recipe ' + noOfDraftRecipes; 
+            draftRecipe.Description__c = 'Test Recipe';
+            noOfDraftRecipes++; 
+            testDraftRecipes.add(draftRecipe);
+
+        }
+        insert testDraftRecipes; 
+
+        List<Recipe__c> testPublishedRecipes = new List<Recipe__c>();
+        integer noOfPublishedRecipes = 0; 
+        while(noOfPublishedRecipes < 150){
+            Recipe__c publishedRecipe = new Recipe__c();
+            publishedRecipe.Name = 'Published Recipe ' + noOfPublishedRecipes;
+            publishedRecipe.Description__c = 'Published Recipe';
+            publishedRecipe.Active_Time__c = noOfPublishedRecipes;
+            publishedRecipe.Active_Time_Units__c = 'Minutes';
+            publishedRecipe.Servings__c = noOfPublishedRecipes;
+            noOfPublishedRecipes++; 
+            testPublishedRecipes.add(publishedRecipe);
+        }
+        insert testPublishedRecipes; 
+
+        List<Cookbook__c> testCookbooks = new List<Cookbook__c>();
+        integer noOfCookbooks = 0; 
+        while(noOfCookbooks < 10){
+            Cookbook__c cookbookRec = new Cookbook__c();
+            cookBookRec.Name = 'Test Cookbook ' + noOfCookbooks; 
+            noOfCookbooks++; 
+            testCookbooks.add(cookbookRec);
+        }
+
+        insert testCookbooks; 
+
+        List<Recipe_Usage__c> testRecipeUsageRecords = new List<Recipe_Usage__c>();
+        integer recipeIndex = 0; 
+
+
+        for(Cookbook__c cookbookRecord : testCookbooks){
+            Recipe_Usage__c testRecipeUsageRecord = new Recipe_Usage__c();
+            testRecipeUsageRecord.Cookbook__c = cookbookRecord.Id; 
+            testRecipeUsageRecord.Recipe__c = testDraftRecipes[recipeIndex].Id;
+            testRecipeUsageRecords.add(testRecipeUsageRecord);
+            recipeIndex++;
+        }
+
+        insert testRecipeUsageRecords; 
+
+
+
+   }
+   @isTest public static void validateSetAsDraftOnInsert(){
+       Recipe__c recipeRecord = [SELECT ID, Draft__c FROM Recipe__c WHERE Draft__c = TRUE LIMIT 1];
```

Comment:
For these insert tests - generally, you would create the record in the test, then retrieve and check the value of the record you just inserted. Otherwise, by querying for a record that meets the criteria then asserting the criteria, you are guaranteeing the test will **always** pass, even if the code doesn't quite work.